ansibleapp-base/Dockerfile

```diff
@@ -8,11 +8,11 @@ RUN cd /tmp && tar zvxf ./oc.tar.gz -C /usr/bin --strip-components=1
 RUN yum install -y net-tools bind-utils wget epel-release
 RUN yum install -y ansible
 RUN echo "localhost ansible_connection=local" >> /etc/ansible/hosts
+RUN sed -i 's/#roles_path.*$/roles_path = \/opt\/ansible\/roles/' /etc/ansible/ansible.cfg
```

Comment:
just for readability, if you want, you can use a sed separator other than '/'. like

```
sed -i 's|#roles_path.*$|roles_path = /opt/ansible/roles|'
```